content/source/docs/enterprise/workspaces/variables.html.md

```diff
@@ -62,12 +64,13 @@ HCL can be used for Terraform variables, but not for environment variables. The
 
 Terraform often needs cloud provider credentials and other sensitive information that shouldn't be widely available within your organization.
 
-To protect these secrets, you can mark any any Terraform or environment variable as sensitive data by clicking its "Sensitive" checkbox in edit mode.
+To protect these secrets, you can mark any any Terraform or environment variable as sensitive data by clicking its "Sensitive" checkbox (visible when editing).
+
+Marking a variable as sensitive prevents anybody (including you) from viewing its value in TFE's UI or API.
 
-Marking a variable as sensitive severely limits how users can interact with it:
+Users with edit permissions can set a new name or value for sensitive variables, and can change whether the value is treated as HCL. However, no one can change a sensitive variable into a non-sensitive one. (If a variable was accidentally marked sensitive, you can delete it and create a new variable to replace it.)
```

Comment:
Mm, I think being able to _replace_ the value makes sense, and switching to HCL makes sense as part of replacing the value. Agree that it would be demented to do that sight-unseen without setting a new value. 

Oh, and if you're hitting ppl up w/ feedback anyway, I also think HCL variables ought to have indicators visible outside of editing mode.